js/app/conf/signature_type.js

```diff
@@ -357,58 +357,87 @@ define([], () => {
 
     // all k-space exits are static or K162
     let c1WH = {
-        1: 'H121 - C1',
-        2: 'C125 - C2',
-        3: 'O883 - C3',
-        4: 'M609 - C4',
-        5: 'L614 - C5',
-        6: 'S804 - C6',
-        7: 'F353 - C12 Thera'
+        1:  'H121 - C1',
+        2:  'C125 - C2',
+        3:  'O883 - C3',
+        4:  'M609 - C4',
+        5:  'L614 - C5',
+        6:  'S804 - C6',
+        7:  'N110 - H',
+        8:  'J244 - L',
+        9:  'Z060 - 0.0',
+        10: 'F353 - C12 Thera'
     };
 
     // all w-space -> w-space are statics or K162
     let c2WH = {
-        1: 'Z647 - C1',
-        2: 'D382 - C2',
-        3: 'O477 - C3',
-        4: 'Y683 - C4',
-        5: 'N062 - C5',
-        6: 'R474 - C6',
-        7: 'F135 - C12 Thera'
+        1:  'Z647 - C1',
+        2:  'D382 - C2',
+        3:  'O477 - C3',
+        4:  'Y683 - C4',
+        5:  'N062 - C5',
+        6:  'R474 - C6',
+        7:  'B274 - H',
+        8:  'A239 - L',
+        9:  'E545 - 0.0',
+        10: 'F135 - C12 Thera'
     };
 
     // all k-space exits are static or K162
     let c3WH = {
-        1: 'V301 - C1',
-        2: 'I182 - C2',
-        3: 'N968 - C3',
-        4: 'T405 - C4',
-        5: 'N770 - C5',
-        6: 'A982 - C6',
-        7: 'F135 - C12 Thera'
+        1:  'V301 - C1',
+        2:  'I182 - C2',
+        3:  'N968 - C3',
+        4:  'T405 - C4',
+        5:  'N770 - C5',
+        6:  'A982 - C6',
+        7:  'D845 - H',
+        8:  'U210 - L',
+        9:  'K346 - 0.0',
+        10: 'F135 - C12 Thera'
     };
 
     // no *wandering* w-space -> w-space
     // all holes are statics or K162
     let c4WH = {
-        1: 'S047 - H',
-        2: 'N290 - L',
-        3: 'K329 - 0.0'
+        1:  'P060 - C1',
+        2:  'N766 - C2',
+        3:  'C247 - C3',
+        4:  'X877 - C4',
+        5:  'H900 - C5',
+        6:  'U574 - C6',
+        7:  'S047 - H',
+        8:  'N290 - L',
+        9:  'K329 - 0.0',
+        10: 'P060 - C12 Thera'
```

Comment:
Where did you get this information? - http://anoik.is/wormholes does not list a "Thera" wh for (C4, C5, C6). `P060 ` is a C4 -> C1 hole according to the table. 
```suggestion
        9:  'K329 - 0.0'
```